### PR TITLE
Feature branch login/register into development 

### PR DESCRIPTION
**Pull request by FelixAlexK**
Backend implemented, now you can create a new user account and log in

### CODE REVIEW

File: app/src/main/java/com/example/maraudersmap/LoginActivity.kt [hunk 1]
```diff
@@ -91,4 +108,50 @@ class LoginActivity : AppCompatActivity() {
...
     )
+
+    /**
+     * validates input string
+     * @param inputString String to validate
+     * @return True if input is valid
+     */
+    private fun validateInput(inputString: String): Boolean{
+
+        if(inputString.isBlank() || inputString.isBlank()){
```
[the remark this answers is not in the document]
Comment (FelixAlexK) [reply app/src/main/java/com/example/maraudersmap/LoginActivity.kt]: Good question 🤔 I Fixed it

File: app/src/main/java/com/example/maraudersmap/RegisterActivity.kt [hunk 2]
```diff
@@ -68,9 +103,69 @@ class RegisterActivity : AppCompatActivity() {
...
     }
+
+    /**
+     * validates input string
+     * @param inputString String to validate
+     * @return True if input is valid
+     */
+    private fun validateInput(inputString: String): Boolean{
+
+        if(inputString.isBlank() || inputString.isBlank()){
```
Comment (julianertle) [inline app/src/main/java/com/example/maraudersmap/RegisterActivity.kt]: why is inputString.isBlank()) checked twice?

Comment (FelixAlexK) [reply app/src/main/java/com/example/maraudersmap/RegisterActivity.kt]: I already fixed it

**Review by julianertle — COMMENTED** [review]
Looking good for now but check my 2 comments

**Review by julianertle — APPROVED** [review]
Looking good after duplication removal

File: app/src/main/java/com/example/maraudersmap/LoginActivity.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (julianertle) [reply app/src/main/java/com/example/maraudersmap/LoginActivity.kt]: You're the goat @FelixAlexK

**Pull request merged**